package.json

```diff
@@ -12,13 +12,19 @@
     "test": "jest --coverage",
     "test:watch": "jest --coverage --watch",
     "commit": "git cz",
+    "prerelease": "git checkout master && git pull origin master && npm i && validate",
```

Comment:
Where is this and `npm run  postrelease` and `npm run validate` being called? Is that something "standard-version" does behind the scenes?

Reply:
That's an NPM convention; you can actually `pre[anything]` and `post[anything]` and they'll automatically get called before/after you `npm run [anything]`.

https://docs.npmjs.com/misc/scripts ... not too far down the page when it's talking about "arbitrary scripts." Super-duper handy!

Reply:
Wow. That is awesome!